addon/components/search-help-modal/component.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
```

Comment:
In general, we try to add basic docstrings to all new components (what they are, what they do, what params they need). I tend to encourage this much more strongly in ember-osf, simply because we intend this repo to be highly reusable shared code.